### ISSUE TITLE
Change a part of a 3D model to a new entity which is loaded from a local or remote path

### ISSUE THREAD

**Issue by whereisnothing** [google/filament#5916]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

**Describe the solution you'd like**
First we show a 3D glb model which is composed of many parts.Then take hair for example,we now load a new part(ie. hair) from a local path on Android phone or a remote path on server which contains a changed part of the model.At last we change the original part (ie. hair) with the new hair loaded from the new path.How can we achieve this?Thanks very much.

**Describe alternatives you've considered**

**OS and backend**
Is this feature OS/backend specific? If so, explain why.

**Comment by pixelflinger** [comment]
why can't you do this manually?

**Comment by prideout** [comment]
Yes, like @pixelflinger said, this could be done manually by simply removing the old hair from the Scene and adding the new hair.  gltfio exposes all the underlying Filament primitives, so anything is possible.  It gets a little trickier if the hair is part of a skinning system, but gltfio provides attachSkin / detachSkin methods to make this easier.

Alternatively, you could split up your original glb, so that the hair is separate from the other parts of the model.
